tests/test_oauth2_validators.py

```diff
@@ -236,7 +236,7 @@ def test_save_bearer_token__with_new_token_equal_to_existing_token__revokes_old_
 
         self.validator.save_bearer_token(token, self.request)
 
-        self.assertEqual(1, RefreshToken.objects.count())
+        self.assertEqual(2, RefreshToken.objects.count())
```

Comment:
It's unclear to me why this value changed?

Reply:
Since the refresh token is no longer deleted on refresh (just marked revoked), the count increases here. We could also write this: `self.assertEqual(1, RefreshToken.objects.filter(revoked__isnull=True).count())`

Reply:
Yeha, could you add that in the test? Then I'll land 👌

Reply:
Updated. Thanks!